ddl/src/main/java/com/adaptivescale/rosetta/ddl/targets/bigquery/BigQueryDDLGenerator.java

```diff
@@ -55,6 +55,10 @@ public String createTable(Table table, boolean dropTableIfExists) {
             builder.append("`").append(table.getName()).append("`; \n");
         }
 
+        if (table.getSchema() != null && !table.getSchema().isBlank()) {
```

Comment:
wouldn't this create repetitive "CREATE SCHEMA IF NOT EXISTS" statement for each table instead of once per DB?